.github/workflows/integration_test.yml

```diff
@@ -26,7 +26,10 @@ jobs:
         run: |
           export CLUSTER="chart-testing"
           hack/local-up-chaos-mesh.sh
-
+          kubectl set env deployment/chaos-dashboard SECURITY_MODE=true -n chaos-testing
+          sleep 5
```

Comment:
Can u explain why it needs to sleep for 5s? I am a little confused about this.

Reply:
before add `sleep 5` will get error:

```
an error occurred forwarding 2333 -> 2333: error forwarding port 2333 to pod a98ee83ece52039639db7b1ee0d1811b1380ad10e5bc97b3dce3cdfb5de80e2b, uid : failed to find sandbox "a98ee83ece52039639db7b1ee0d1811b1380ad10e5bc97b3dce3cdfb5de80e2b" in store: does not exist
```

maybe the pod is not finished the upgrade, so just sleep some seconds

Reply:
Got it!